### PR TITLE
[ALLI-7570] Add support for marking users and lists protected.

### PR DESCRIPTION
**Pull request by EreMaijala**
Any protection prevents an account from expiring and expiration messages from being sent.

### CODE REVIEW

File: module/FinnaConsole/src/FinnaConsole/Command/Lists/ProtectedHandlerFactory.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Factory for the "list protected users" task.
```
Comment (LuomaJuha) [inline module/FinnaConsole/src/FinnaConsole/Command/Lists/ProtectedHandlerFactory.php]: Factory for the "list protected lists" task?

File: module/FinnaConsole/src/FinnaConsole/Command/Lists/ProtectedHandlerFactory.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+namespace FinnaConsole\Command\Lists;
+
+use Laminas\ServiceManager\Exception\ServiceNotCreatedException;
+use Laminas\ServiceManager\Exception\ServiceNotFoundException;
+use Laminas\ServiceManager\Factory\FactoryInterface;
+use Psr\Container\ContainerExceptionInterface as ContainerException;
+use Psr\Container\ContainerInterface;
+
+/**
+ * Factory for the "list protected users" task.
```
Comment (LuomaJuha) [inline module/FinnaConsole/src/FinnaConsole/Command/Lists/ProtectedHandlerFactory.php]: Factory for the "list protected lists" task?

File: module/FinnaConsole/src/FinnaConsole/Command/Lists/ListProtected.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+{
+    /**
+     * The name of the command (the part after "public/index.php")
+     *
+     * @var string
+     */
+    protected static $defaultName = 'lists/list_protected';
+
+    /**
+     * User table
```
Comment (LuomaJuha) [inline module/FinnaConsole/src/FinnaConsole/Command/Lists/ListProtected.php]: Userlist table?

**Review by LuomaJuha — CHANGES_REQUESTED** [review]
Pari kommenttia löytyi jotka taitavat olla jääneet copypasten tuloksena.

**Comment by EreMaijala** [comment]
@LuomaJuha Joo, kommentit fiksattu.

**Review by LuomaJuha — APPROVED** [review]

**Pull request merged**